java/src/jmri/jmrix/can/CanMessage.java

```diff
@@ -108,125 +127,123 @@ public int hashCode() {
 
     /**
      * Note that a CanMessage and a CanReply can be tested for equality
+     * @param a CanMessage or CanReply to test against
      */
     @Override
     public boolean equals(Object a) {
-        if (a == null) {
-            return false;
-        }
-        // check for CanFrame equality, that's sufficient
-        if (a instanceof CanFrame) {
-            CanFrame m = (CanFrame) a;
-            if ((_header != m.getHeader()) || (_isRtr != m.isRtr()) || (_isExtended != m.isExtended())) {
-                return false;
-            }
-            if (_nDataChars != m.getNumDataElements()) {
-                return false;
-            }
-            for (int i = 0; i < _nDataChars; i++) {
-                if (_dataChars[i] != m.getElement(i)) {
-                    return false;
-                }
-            }
-            return true;
-        } else {
-            return false;
-        }
+        return isEqual(this,a);
     }
 
     /**
      * {@inheritDoc}
-     * default toString does not contain the header
-     * this format matches @CanReply
+     * This format matches @CanReply
      */
     @Override
     public String toString() {
-        String s = String.format("[%x] ", _header);
-        for (int i = 0; i < _nDataChars; i++) {
-            if (i != 0) {
-                s += " ";
-            }
-            s = StringUtil.appendTwoHexFromInt(_dataChars[i] & 255, s);
-        }
-        return s;
+        return getToString();
     }
 
+    /**
+     * {@inheritDoc}
+     * This format matches @CanReply
+     */
     @Override
     public String toMonitorString() {
-        StringBuffer buf = new StringBuffer();
-        buf.append("(" + Integer.toHexString(getHeader())
-                + (isExtended() ? " ext)" : ")"));
-        for (int i = 0; i < getNumDataElements(); i++)
-        {
-            buf.append(" " + jmri.util.StringUtil.twoHexFromInt(getElement(i)));
-        }
-	return buf.toString();
+        return monString();
     }
 
+    /**
+     * {@inheritDoc}
+     * Always False
```

Comment:
Change `Always False` to `@return always false`; this will cause the return statement in the final JavaDoc to be overridden from the inherited documentation.

Reply:
There are a few other places where this will be useful, thanks.